### ISSUE TITLE
Missing types

### ISSUE THREAD

**Issue by antoniogiroz** [hairyf/overlays#12]
I've installed the `0.7.3` version in a vue project with typescript, and I get this error importing the package `'@unoverlays/vue`:

```
Could not find a declaration file for module '@unoverlays/vue'. '~/demo/node_modules/.pnpm/@unoverlays+vue@0.7.3_vue@3.2.47/node_modules/@unoverlays/vue/index.esm.js' implicitly has an 'any' type.
  There are types at '~/demo/node_modules/@unoverlays/vue/index.d.ts', but this result could not be resolved when respecting package.json "exports". The '@unoverlays/vue' library may need to update its package.json or typings.ts(7016)
```

**Comment by hairyf** [comment]
You can use 0.7.2 first, and I will fix it

**Comment by hairyf** [comment]
Fixed in 0.7.4
